.concourse/pipeline.yaml.gomplate

```diff
@@ -346,10 +361,15 @@ jobs:
       file: output/kubecf-bundle-v*.tgz
       acl: public-read
 
-- name: deploy-diego
+{{- range $_, $cfScheduler := slice "diego" "eirini" }}
+
+- name: deploy-{{ $cfScheduler }}
   max_in_flight: 2
```

Comment:
```suggestion
  max_in_flight: 1
```

With this change, we don't make the deployment wait for a lock, but they will queue up. The deployment won't happen anymore in parallel, but the test runs will.